openlibrary/components/MergeUI.vue

```diff
@@ -1,8 +1,13 @@
 <template>
   <div id="app">
-    <MergeTable :olids="url.searchParams.get('records', '').split(',')" ref="mergeTable"/>
-
+    <MergeTable
+        :olids="url.searchParams.get('records', '').split(',')"
+        :show_diffs="show_diffs"
+        ref="mergeTable"
+        />
     <button @click="doMerge" :disabled="mergeStatus == 'Saving...'">Do Merge</button>
+    <div id="diffs-toggle"><input type="checkbox" id="diff-checkbox" title="Show textual differences" v-model="show_diffs" />
+    <label for="diff-checkbox">Show text diffs</label></div>
```

Comment:
Placing the checkbox in the `label` lets us avoid the `id` :)
```suggestion
    <div id="diffs-toggle">
        <label>
            <input type="checkbox" title="Show textual differences" v-model="show_diffs" />
            Show text diffs
        </label>
    </div>
```